processing/src/main/java/org/apache/carbondata/processing/store/writer/CarbonFactDataWriterImplForIntIndexAndAggBlock.java

```diff
@@ -250,6 +254,7 @@ public NodeHolder buildDataNodeHolder(IndexStorage<int[]>[] keyStorageArray, byt
     }
     long blockletDataSize =
         holder.getKeyArray().length + holder.getDataArray().length + indexBlockSize;
+    LOGGER.info("A new blocklet is added, its data size is: " + blockletDataSize + " Byte");
```

Comment:
Move this line after call to writeDataToFile, because if writeDataToFile fails, we should not log that a blocklet is added.